Dockerfile

```diff
@@ -22,15 +44,15 @@ ENV DEPS \
     bash \
```

Comment:
Would you add a mention of why we need `bash` to the comment block since we explicitly say why we need `redis-tools`?

Reply:
Please see commit d1d67c0.